components/core/src/os/users/linux.rs

```diff
@@ -40,3 +40,7 @@ pub fn get_effective_uid() -> u32 {
 pub fn get_home_for_user(username: &str) -> Option<PathBuf> {
     linux_users::get_user_by_name(username).map(|u| PathBuf::from(u.home_dir()))
 }
+
+pub fn root_level_account() -> String {
+    "root".to_string()
```

Comment:
Since the windows implementation returns `env::var("COMPUTERNAME").unwrap().to_uppercase() + "$"` which is actually a `String` I'd have to convert one or the other.